### PR TITLE
⭐ Added My Nintendo

### PR DESCRIPTION
**Pull request by ACertainCoder**
**What's the name of the service?**
My Nintendo

**What should the Presence display?**
Informations on what you are currently doing on My Nintendo

**Why do you want support for this service?**
If you are using Nintendo's consoles quiet a lot, you can get many benefits like discounts just by playing / buying stuff.

**Are you able to provide an image for the service? (512x512 minimum)**
logo_big

**What should the presence look like when active?**
grafik
grafik
grafik

### CODE REVIEW

File: My Nintendo/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "name": "ACertainCoder",
+    "id": "320507572455079949"
+  },
+  "service": "My Nintendo",
+  "description": {
+    "en": "Thanks to My Nintendo makes playing software and interacting with Nintendo even more fun. You can earn points and get rewards such as discounts and additional content for games.",
+    "de": "Dank My Nintendo macht das Spielen von Software und die Interaktion mit Nintendo noch mehr Spaß. Du kannst Punkte sammeln und Belohnungen wie Rabatte und Zusatzinhalte für Spiele erhalten."
+  },
+  "url": [
+      "my.nintendo.com"
```
Comment (Alanexei) [inline My Nintendo/dist/metadata.json]: Make this a string not an array

File: My Nintendo/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  ],
+  "version": "1.0",
+  "logo": "https://i.imgur.com/eywFbXQ.png",
+  "thumbnail": "https://i.imgur.com/ekWK1iw.png",
+  "color": "#e60012",
+  "tags": ["my", "nintendo", "switch", "ds", "2ds", "3ds"],
```
Comment (Alanexei) [inline My Nintendo/dist/metadata.json]: Remove the "my" tag

File: My Nintendo/dist/metadata.json (hunk 1 above)
Comment (Alanexei) [reply My Nintendo/dist/metadata.json]: I am talking about the url btw

Comment (ACertainCoder) [reply My Nintendo/dist/metadata.json]: Done 👍

File: My Nintendo/dist/metadata.json (hunk 2 above)
Comment (ACertainCoder) [reply My Nintendo/dist/metadata.json]: Done 👍

File: My Nintendo/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "service": "My Nintendo",
+  "description": {
+    "en": "Thanks to My Nintendo makes playing software and interacting with Nintendo even more fun. You can earn points and get rewards such as discounts and additional content for games.",
+    "de": "Dank My Nintendo macht das Spielen von Software und die Interaktion mit Nintendo noch mehr Spaß. Du kannst Punkte sammeln und Belohnungen wie Rabatte und Zusatzinhalte für Spiele erhalten."
+  },
+  "url": "my.nintendo.com",
+  "version": "1.0",
+  "logo": "https://i.imgur.com/eywFbXQ.png",
+  "thumbnail": "https://i.imgur.com/ekWK1iw.png",
```
Comment (Alanexei) [inline My Nintendo/dist/metadata.json]: Try and find a higher quality thumbnail.

File: My Nintendo/dist/metadata.json (hunk 2 above)
Comment (doomlerd) [reply My Nintendo/dist/metadata.json]: if you need to add two words in a tag next time just put a dash between them : "my-nintendo"

File: My Nintendo/dist/metadata.json (hunk 3 above)
Comment (doomlerd) [reply My Nintendo/dist/metadata.json]: It's fine.

File: My Nintendo/dist/metadata.json (hunk 2 above)
Comment (ACertainCoder) [reply My Nintendo/dist/metadata.json]: Ok, thanks

File: My Nintendo/dist/metadata.json (hunk 3 above)
Comment (Bas950) [reply My Nintendo/dist/metadata.json]: Looks good to me..

**Pull request merged**